lib/modules/platform/codecommit/index.md

```diff
@@ -38,10 +48,46 @@ Let Renovate use AWS CodeCommit authentication keys by doing one of the followin
   --token: AWS session token, if you have one
   ```
 
-## AWS IAM security policies
-
-- Make sure to attach the [AWSCodeCommitFullAccess](https://docs.aws.amazon.com/codecommit/latest/userguide/security-iam-awsmanpol.html#managed-policies-full) policy to your IAM User
-- It is recommended to also attach the [IAMReadOnlyAccess](https://docs.aws.amazon.com/IAM/latest/UserGuide/security-iam-awsmanpol.html) policy to your IAM User
+## AWS IAM permission policy
+
+Create a new policy for renovate with these permissions,
+Then attach it to the user/role.
+
+Add `"iam:GetUser",` to the `Action` array if you are using `Authentication as a USER` it will help reduce AWS requests
+Change the `Resource` to the resources you want to use
```

Comment:
Reformat according to our convention:
- One sentence per line
- Sentences end in periods (.)

Reply:
check now, is that good?